app/views/hostgroups/index.html.erb

```diff
@@ -16,6 +18,14 @@
           <%= label_with_link(hostgroup, 150, authorizer) %>
         </td>
         <td>
+          <%= link_to hostgroup.hosts_count, hosts_path(:search => "hostgroup_fullname = #{hostgroup}") %>
+        </td>
+        <td>
+          <% if hostgroup.children? %>
```

Comment:
@shlomizadok I think this is actually a case of @domcleal agreeing with me :wink: